portfolio/src/main/webapp/index.html

```diff
@@ -17,7 +17,27 @@ <h1> Alisha Kewalramani </h1>
            </ul>
       </nav>
       <div class="alt1">
-        <img src="images/alisha.jpeg"/>
+        <div class="slideshowContainer">
+            <div class="individualSlide">
+                <img src="images/alisha.jpeg"/>
+            </div>
+            <div class="individualSlide">
+                <img src="images/solopic.JPG"/>
+            </div>
+            <div class="individualSlide">
+                <img src="images/solopic2.jpg"/>
+            </div>
+            <div class="individualSlide">
+                <img src="images/solopic3.JPG"/>
+            </div>
+         </div>
+        <div class="dotContainer">
+            <span class="dot" onclick="currentSlide(1)"></span>
+            <span class="dot" onclick="currentSlide(2)"></span>
+            <span class="dot" onclick="currentSlide(3)"></span>
+            <span class="dot" onclick="currentSlide(4)"></span>
```

Comment:
looks like currentSlide isn't defined anywhere?

also, i'd rename to include a verb, e.g. showSlide(i)